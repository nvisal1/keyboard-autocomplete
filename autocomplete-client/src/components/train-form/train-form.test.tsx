import React from 'react';
import renderer from 'react-test-renderer';

import TrainForm, { TrainFormProps } from './train-form';

describe('When the TrainForm component is created', () => {
    it('should render correctly', () => {
        const requiredProps = getRequiredProps();
        const currentSnapshot = renderer.create(<TrainForm { ...requiredProps }></TrainForm>).toJSON();

        expect(currentSnapshot).toMatchSnapshot();
    });
});

function getRequiredProps(): TrainFormProps {
   const handleSubmit = (_: string) => {}; 

   return { handleSubmit };
}