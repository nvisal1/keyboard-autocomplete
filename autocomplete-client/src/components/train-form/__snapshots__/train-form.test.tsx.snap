// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When the TrainForm component is created should render correctly 1`] = `
<form
  className="train-form"
  onSubmit={[Function]}
>
  <textarea
    className="train-form__textarea"
    cols={30}
    name="training-passage"
    rows={10}
  />
  <button
    className="train-form__submit-button"
  >
    Submit
  </button>
</form>
`;
